.gitignore

```diff
@@ -1,2 +1,8 @@
+build/
```

Comment:
I know there is much debate about including `.gitignore` in a repository or not. I'm fine with it, just wanted to make sure it was intended.

Reply:
I don't feel strongly either. I probably tracked it for my own convenience of developing on different machines. But this might be a good time to untrack it.